Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/Oni.yml

```
@@ -20,21 +20,25 @@
   - type: BonusMeleeDamage
     damageModifierSet:
       coefficients:
-        Blunt: 1.35
-        Slash: 1.2
-        Piercing: 1.2
-        Asphyxiation: 1.35
+        Blunt: 1.25
+        Slash: 1.25
+        Piercing: 1.25
+        Asphyixiation: 1.25
```

I don't know how you added the i there :⁾
```suggestion
        Asphyxiation: 1.25
```